region/lowernorfair/east.json

```diff
@@ -537,39 +552,114 @@
               "requires": [
                 {"or": [
                   {"and": [
-                    "canKillYellowSpacePirates",
+                    "heatProof",
+                    {"or": [
+                      "canKillYellowSpacePirates",                          
+                      {"canComeInCharged": {
+                        "fromNode": 2,
+                        "framesRemaining": 120
+                      }}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canHeatRun",
                     {"or": [
                       {"and": [
                         "ScrewAttack",
-                        "SpaceJump"
+                        {"cost": [
+                          {"heatFrames": 250}
+                        ]}
+                      ]},
+                      {"and": [
+                        "Charge",
+                        "Plasma",
+                        {"cost": [
+                          {"heatFrames": 600}
+                        ]}
                       ]},
-                      "canIBJ",
+                      {"cost": [
+                        {"heatFrames": 450},
+                        {"ammo": [{
+                          "type": "Super",
+                          "count": 9
+                        }]}
+                      ]},
+                      {"and": [
+                        "Charge",
+                        "Ice",
+                        "Spazer",
+                        "Wave",
+                        {"cost": [
+                          {"heatFrames": 1150}
+                        ]}
+                      ]},
+                      {"canComeInCharged": {
```

Comment:
I am guessing that this has no associated cost because it's just an indicator that we don't need to kill the pirates, and the spark and travel cost is in the second part of the reqs, where actual travel is addressed.

Reply:
Yes I had to put it in twice to make that structure work.